Client/mods/deathmatch/logic/CResource.h

```diff
@@ -69,6 +69,7 @@ class CResource
     void                 SetResourceEntity(CClientEntity* pEntity) { m_pResourceEntity = pEntity; }
     class CClientEntity* GetResourceDynamicEntity() { return m_pResourceDynamicEntity; }
     void                 SetResourceDynamicEntity(CClientEntity* pEntity) { m_pResourceDynamicEntity = pEntity; }
+    SString              GetResourceDirectoryPath() { return GetResourceDirectoryPath(eAccessType::ACCESS_PUBLIC, ""); }
     SString              GetResourceDirectoryPath(eAccessType accessType, const SString& strMetaPath);
```

Comment:
How about using std::string instead of SString?

Reply:
Oops, yeah forgot about that part.
Sure

Reply:
Okay, fair enough.
I will accept the modifications if you also solve the other conversations, as suggested by TheNormalnij.

Reply:
As far as I see all conflict have been resolved